### ISSUE TITLE
Short transmissions fail

### ISSUE THREAD

**Issue by jetpax** [ggerganov/ggwave#22]
Sending strings less than 4 chars from IOS ggwaveapp to command line (using python binding) fail, 4 chars and above work just fine.

**Comment by ggerganov** [comment]
Thanks for the report and sorry about that - I "broke" it with ggwave `v0.2.0`. The number of ECC bytes now used is determined like this:

https://github.com/ggerganov/ggwave/blob/5b5293d764ddde31dad5fa9f4089bdf7816734e0/src/ggwave.cpp#L248-L250

Before it was determined like this:

https://github.com/ggerganov/ggwave/blob/ff5c56907196c2a060498b465843dfdc09cb303c/src/ggwave.cpp#L244-L246

I decided that using 4 ECC bytes for messages with length 1-3 is too much so I reduced it to 2.

The iOS app is still using ggwave `v0.1.0` so it will not be compatible with `v0.2.0` for length < 4. I will be updating the iOS and Android apps (#23) in the next few days, so after the update it should work for all data lengths.
